.atomist/editors/AddTypeScriptEditor.ts

```diff
@@ -49,10 +49,14 @@ class AddTypeScriptEditor implements EditProject {
     description: string;
 
     edit(project: Project) {
-        if (!IsRugArchive(project) || !IsSetUpForTypeScript(project)) {
+
+        if (!IsRugArchive(project)) {
+            console.log("Can't run when not a rug archive");
             return;
         }
 
+        project.editWith("AddTypeScript", {});
```

Comment:
I'm not sure this is a good idea. If we decide it is, we should made the change to all the AddTypeScript* editors.